src/ui/index.js

```diff
@@ -57,6 +57,8 @@ module.exports = async (kbnServer, server, config) => {
   });
 
   server.decorate('reply', 'renderApp', function (app) {
+    if (_.isString(app)) app = uiExports.getApp(app) || uiExports.getHiddenApp(app);
```

Comment:
This feel unrelated, and I would prefer it didn't work this way. Can we move it out of this pr please?